style.css

```diff
@@ -0,0 +1,198 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap');
+
+html {
+  font-size: 62.5%;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  scroll-behavior: smooth;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+label {
+  display: none;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 1.6rem;
+  background-color: #f5f5f5;
+  overflow-x: hidden;
+  width: 100%;
+  position: relative;
+}
+
+::-webkit-scrollbar {
+  width: 12px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #5e5f5f;
+  border-radius: 6px;
+}
+
+input {
+  outline: none;
+  border: none;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+li {
+  padding: 0;
+}
+
+.bar {
+  position: absolute;
+  bottom: 1rem;
+  height: 5px;
+  width: 135px;
+  border-radius: 10px;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #0d151e;
+}
+
+.btn {
+  display: inline-block;
+  padding: 1rem;
+  border: 1px solid #6070ff;
+  border-radius: 1rem;
+  color: #396df2;
+  text-align: center;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  transition: ease color 2s;
+}
+
+#header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  background-color: #fff;
+  position: fixed;
+  z-index: 999;
+}
+
+#header .bar {
+  display: none;
+}
+
+#header.shadow {
+  box-shadow: -3px -2px 35px 11px rgba(0, 0, 0, 0.15);
+}
+
+#header.open .bar {
+  display: flex;
+}
+
+.logo {
+  padding-left: 2rem;
+}
+
+.logo a {
+  font-weight: 700;
+  font-size: 2rem;
+  color: #6070ff;
+  flex-wrap: nowrap;
+}
+
+.logo:hover {
+  animation-name: rotate;
+  animation-duration: 1.7s;
+}
+
+@keyframes rotate {
+  0% {
+    transform: translateX(0);
+  }
+
+  100% {
+    transform: translateX(36px);
+  }
+}
+
+.navbar {
+  height: 100%;
+}
+
+.nav-list {
+  display: none;
+}
+
+.navbar .menu {
+  height: 100%;
+  line-height: 50px;
+  padding-right: 2.5rem;
+}
+
+.navbar .menu .bx {
+  font-size: 3rem;
+  color: #6070ff;
+  line-height: 50px;
+}
+
+.main-container {
+  width: 100%;
+  height: 100%;
+  z-index: 999;
+}
+
+.caption {
+  width: 100%;
+  height: calc(718px - 60px);
+  background-image: url("background-Image.png");
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  overflow: hidden;
+  border-bottom-left-radius: 5rem;
+  background-color: #fff;
+  position: relative;
+}
```

Comment:
- Kindly make sure the background image is applied to all containers except the  nav bar, It could be better to move it to the main container class so that it will be applied for all pages except  the navbar :+1: